README.md

```diff
@@ -28,24 +28,29 @@ The current multibase table is [here](multibase.csv):
 
 ```
 encoding      codes   name
-base1         1       unary tends to be 11111
-base2         0       binary has 1 and 0
-base8         7       highest char in octal
-base10        9       highest char in decimal
-base16        F, f    highest char in hex
-base32        U, u    rfc4648 - highest letter
-base32hex     V, v    rfc4648 - highest char
-base58flickr  Z       highest char
-base58btc     z       highest char
-base64        y       rfc4648 highest char
-base64url     Y       rfc4648 highest char
+base1,        1,    unary tends to be 11111
+base2,        0,    binary has 1 and 0
+base8,        7,    highest char in octal
+base10,       9,    highest char in decimal
+base16,       F, f, highest char in hex
+base32hex,    V, v, rfc4648 no padding - highest char
+base32hexpad, T, t, rfc4648 with padding
+base32,       B, b, rfc4648 no padding
+base32pad,    C, c, rfc4648 with padding
+base32z,      h,    z-base-32 - used by Tahoe-LAFS - highest letter
+base58flickr, Z,    highest letter
+base58btc,    z,    highest letter
+base64,       m,    rfc4648 no padding
+base64pad,    M,    rfc4648 with padding - MIME encoding
+base64url,    u,    rfc4648 no padding
+base64urlpad, U,    rfc4648 with padding
+binary,       X,    8-bit binary (encoder and decoder keeps data unmodified)
 ```
 
 These encodings are being considered:
 
 ```
 base128
-base256ascii  X       ascii
```

Comment:
Sorry, @jbenet, but I do not see what you mean. Not every encoding can be encoded as ASCII, because that only has 127 fixed runes (go-lang term) and in [extended ASCII](https://en.wikipedia.org/wiki/Extended_ASCII) some extra used by Western languages. There are no codepoints (Unicode term) in that for most of the languages (like for the "ő" "Ő" "ű" "Ű" Hungarian graphemes). If you want to include EBCDIC, codepages, Windows 1252 encoding and other pre-Unicode encodings, you will run out of the 127 characters for the current implementations of the varint prefix.

The prefix '0' is implemented in [multiformats/js-multibase](https://github.com/multiformats/js-multibase/blob/0a0bcf437773a712df9568f7a3b7a8ec0e7d9489/src/constants.js#L8) as an encoding that encodes a single byte as 8 characters: 'a' as "01100001"

My goal is to do include the zero encoding or identity encoding, which prefix should I use for that in the multiformats/go-multibase#6 pull request?